### PR TITLE
Feature/rad 284 save notifications in db

### PR DESCRIPTION
**Pull request by b-dabrowski**

### CODE REVIEW

File: services/notifications/src/app/features/notifications/models/notification.model.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  @Column({ nullable: false })
+  chanel: string;
+
+  @Column({ nullable: false })
+  message: string;
+
+  @CreateDateColumn({ type: "timestamp" })
+  createdAt: Date;
+
+  @UpdateDateColumn({ type: "timestamp" })
```
Comment (mkopa) [inline services/notifications/src/app/features/notifications/models/notification.model.ts:42]: Notifications will never be updated

File: services/notifications/src/app/features/notifications/models/notification.model.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    });
+
+    return entities;
+  }
+
+  @PrimaryGeneratedColumn("uuid")
+  id: string;
+
+  @Column({ nullable: false })
+  chanel: string;
```
Comment (mkopa) [inline services/notifications/src/app/features/notifications/models/notification.model.ts:34]: Create an index for the values ​​in this column.
`channel` is a string: userId (from token),` default-all`, `default-authorized`, or default-unauthorized`.

File: services/notifications/src/app/features/notifications/models/notification.model.ts (hunk 1 above)
Comment (b-dabrowski) [reply services/notifications/src/app/features/notifications/models/notification.model.ts]: I have removed it :)

File: services/notifications/src/app/features/notifications/models/notification.model.ts (hunk 2 above)
Comment (b-dabrowski) [reply services/notifications/src/app/features/notifications/models/notification.model.ts]: I have added index 👍

**Pull request merged**